File: C Functions Task/square.java
public class Main{
public static void doSquare(int a,int b)
{
  int result;
  result=(a*a)+(b*b)+(2*(a*b));
  System.out.printf("%d\n",result);
}
public static void main(String[]args)
{
  doSquare(3,5);
  doSquare(2,4);
}
}

File: C Functions Task/todo2/square.c
#include <stdio.h>
void getSq(int a,int b)
{
  int result;
  result=(a*a)+(b*b)+(2*(a*b));
  printf("%d\n",result);
}
int main()
{
    getSq(5,4);
    getSq(6,8);
    return 0;
}